### PR TITLE
VC-2861 Editor can't save with multiple images

### PR DESCRIPTION
**Pull request by dmytro-kovalov**

### CODE REVIEW

**Comment by OlegApanovich** [comment]
Php code style breaking

File: visualcomposer/Helpers/Image.php [hunk 1]
```diff
@@ -203,4 +229,209 @@ protected function getImageAttributes($isLazyload, $newSrc, $src, $attributes, $
...
+        // Calculate retina size
+        if ($resizedWidth * 2 <= $originalWidth) {
+            $retinaImage = $resizedWidth * 2;
+            $sizes['2x'] = (int)$retinaImage;
+        }
+
+        return $sizes;
+    }
+
+    public function getImages($sizes, $image, $imageData)
```
Comment (OlegApanovich) [inline visualcomposer/Helpers/Image.php:302]: Let's please add docblocks for all newly created methods

File: visualcomposer/Helpers/Image.php [hunk 2]
```diff
@@ -203,4 +229,209 @@ protected function getImageAttributes($isLazyload, $newSrc, $src, $attributes, $
...
+                $key = array_search($image['width'], $sizes, true);
+                $images[ $key ] = $dirname . $image['file'];
+            }
+        }
+
+        return $images;
+
+    }
+
+    public function resizeImages($sizes, $image, $imageData)
```
Comment (OlegApanovich) [inline visualcomposer/Helpers/Image.php:380]: Let's please add docblocks for all newly created methods

File: visualcomposer/Helpers/Image.php (hunk 1 above)
Comment (dmytro-kovalov) [reply visualcomposer/Helpers/Image.php:334]: Done

**Review by OlegApanovich — APPROVED** [review]

**Pull request merged**